docs/operation/cluster_and_service_configuration.rst

```diff
@@ -30,12 +31,16 @@ MetalK8s installation.
 
 To add a new static user, perform the following operations:
 
-#. Generate a password ``hash``. The command below requires **apache-utils**
-   package to be installed.
+   .. _Generate-password-hash:
+
+#. Generate a bcrypt hash of your new password.
+
+   - To generate the bcrypt hash, On the Bootstrap node, run the following
+     command while replacing the <password> field with your plaintext password.
 
    .. code-block:: shell
 
-      root@bootstrap $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+      root@bootstrap $ htpasswd -bnBC 14 "" <password> | tr -d ':'
```

Comment:
I thought we said the best thing is to never input passwords in clear text on the CLI... So, better use the suggestion from Nicolas:

```suggestion
      root@bootstrap $ htpasswd -nBC 14 "" | tr -d ':'
      New password:
      Re-type new password:
      <YOUR PASSWORD HASH HERE>
```